content/pl/concepts/_index.md

```diff
@@ -0,0 +1,70 @@
+  * **[kubelet](/docs/admin/kubelet/)**, który komunikuje się z Kubernetes Master.
+  * **[kube-proxy](/docs/admin/kube-proxy/)**, proxy sieciowe pośredniczące w usługach sieciowych Kubernetes.
+
+## Obiekty Kubernetes
+
+Kubernetes składa się z różnych abstrakcyjnych obiektów, które reprezentują stan systemu: wdrożone aplikacje i zadania w kontenerach, powiązane zasoby sieciowe i dyskowe oraz inne informacje o tym, co się dzieje na klasterze. Te abstrakcyjne obiekty są reprezentowane przez API Kubernetes. [Understanding Kubernetes Objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) zawiera więcej szczegółów na ten temat.
+
+Do podstawowych obiektów Kubernetes należą:
+
+* [Pod](/docs/concepts/workloads/pods/pod-overview/)
+* [Service *(Serwis)*](/docs/concepts/services-networking/service/)
+* [Volume *(Wolumin)*](/docs/concepts/storage/volumes/)
+* [Namespace](/docs/concepts/overview/working-with-objects/namespaces/)
+
+Kubernetes zawiera także obiekty abstrakcyjne wyższego poziomu, zbudowane z obiektów podstawowych przy wykorzystaniu [Kontrolerów](/docs/concepts/architecture/controller/), które dostarczają dodatkowe funkcjonalności i udogodnienia. Należą do nich:
+
+ * [Deployment](/docs/concepts/workloads/controllers/deployment/)
+ * [DaemonSet](/docs/concepts/workloads/controllers/daemonset/)
+ * [StatefulSet](/docs/concepts/workloads/controllers/statefulset/)
+ * [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+ * [Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+
+## Warstwa sterowania (*Kubernetes Control Plane*)
+
+Różne komponenty warstwy sterowania, takie jak: *Kubernetes Master* czy *kubelet*, odpowiadają za to, jak Kubernetes komunikuje się z klastrem. Warstwa sterowania przechowuje informacje o wszystkich obiektach Kubernetes w systemie i w sposób ciągły steruje ich stanem. W każdej chwili pętle sterowania reagują na zmiany zachodzące w klastrze, prowadząc do tego, aby stan faktyczny wszystkich obiektów odpowiadał stanowi zamierzonemu przez użytkownika.
+
+Przykładowo, kiedy używasz Kubernetes API do stworzenia Deploymentu, podajesz oczekiwany stan systemu. Warstwa sterowania Kubernetes zapisuje stworzenie tego obiektu, a następnie realizuje Twoje polecenie startując wymagane aplikacje i zlecając ich uruchomienie na węzły klastra — w ten sposób powodując, że faktyczny stan klastra jest zgodny ze stanem zamierzonym.
+
+### Kubernetes Master
+
+*Kubernetes Master* odpowiada za utrzymanie klastra w zamierzonym stanie. Za każdym razem, kiedy korzystasz z `kubectl`, łączysz się z węzłem Kubernetes *master* danego klastra.
+
+> "*Master*" odnosi się do zbioru procesów zarządzających stanem klastra. Zazwyczaj procesy te są uruchomione na pojedynczym węźle klastra, który jest  określany jako "master". Master może być też replikowany w celu zagwarantowania wyższej dostępności i niezawodności.
+
+### Węzły (*Kubernetes Nodes*)
```

Comment:
I suggest adding an explicit fragment identifier that covers just the Polish text:
```
### Węzły (*Kubernetes Nodes*) {#węzły)
```

Reply:
Do you mean "(" instead of "{" in your suggestion?

Reply:
Ah, instead of `{#węzły)` I meant to suggest `{#węzły}`. Oops.

Reply:
Custom anchor added.